site/src/pages/WorkspacePage/useResourcesNav.ts

```diff
@@ -0,0 +1,44 @@
+import { WorkspaceResource } from "api/typesGenerated";
+import { useTab } from "hooks";
+import { useCallback, useEffect } from "react";
+
+export const resourceOptionId = (resource: WorkspaceResource) => {
+  return `${resource.type}_${resource.name}`;
+};
+
+export const useResourcesNav = (resources: WorkspaceResource[]) => {
+  const resourcesNav = useTab("resources", "");
```

Comment:
Just making sure: does an empty string just represent that something isn't selected for `useTab`? If not, and it's just a placeholder, I'm wondering if the default value could be based off the first resource

Reply:
That is so true! Idk why I didn't realize this before, ty!